Mixpanel/MPUIImageToNSDictionaryValueTransformer.m

```diff
@@ -95,9 +95,9 @@ - (id)reverseTransformedValue:(id)value
                     NSError *error;
                     NSData *imageData = [NSData dataWithContentsOfURL:imageUrl options:(NSDataReadingOptions)0 error:&error];
                     if (!error) {
-                        image = [UIImage imageWithData:imageData scale:fminf(1.0, [scale floatValue])];
+                        image = [UIImage imageWithData:imageData scale:fminf(1.0, scale.floatValue)];
                         @synchronized(imageCache) {
-                            [imageCache setValue:image forKey:imageDictionary[@"url"]];
+                            imageCache[imageDictionary[@"url"]] = image;
```

Comment:
I believe this will throw an exception on `nil` while `setValue:forKey:` does not.